File: 69_Sqrt(x)/15ms.cpp
class Solution {
public:
  int mySqrt(int x) {
    if(x == 0)
      return 0;
    else if(x == 1 || x == 2 || x == 3)
      return 1;
    else {
      long i;

      for(i = 2; i < x; i++) {
        if(i * i >= x) {
          if(i * i == x)
            return i;
          else
            return (i - 1);
        }
      }
    }

    return 0;
  }
};

File: 128_Longest_Consecutive_Sequence/15ms.java
class Solution {
  public int longestConsecutiveSequence(int[] nums) {
    if(nums.length == 0) return 0;

    Arrays.sort(nums);

    int majorLen = 1, currentLen = 1;

    for (int i = 1; i < nums.length; i++) {
      if(nums[i] == nums[i - 1]) {
        continue;
      } else if(nums[i] == nums[i - 1] + 1) {
        currentLen++;
      } else {
        majorLen = Math.max(majorLen, currentLen);
        currentLen = 1;
      }
    }

    majorLen = Math.max(majorLen, currentLen);

    return majorLen;
  }
}
